File: socialuni-js/packages/socialuni-im/socialuni-im-api/src/model/RO/SocialuniFriendApplyUserRO.ts
import SocialuniUserRO from "@socialuni/socialuni-api-base/src/model/user/SocialuniUserRO";

export default class SocialuniFriendApplyUserRO extends SocialuniUserRO{
    //申请消息
    friendApplyStatus: string = null
    applyMsg: string = null
    createTime: Date = null
}

File: socialuni-server/socialuni-im-api/src/main/java/com/socialuni/social/im/api/model/RO/SocialuniFriendApplyUserRO.java
package com.socialuni.social.im.api.model.RO;

import com.socialuni.social.common.api.model.user.SocialuniUserShowRO;
import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.Date;

@Data
@EqualsAndHashCode(callSuper = true)
public class SocialuniFriendApplyUserRO extends SocialuniUserShowRO {
    // 待同意
    // 添加成功
    // 已同意
    // 已拒绝
    String friendApplyStatus;
    String applyMsg;
    Date createTime;

    public SocialuniFriendApplyUserRO(SocialuniUserShowRO userRO) {
        super(userRO);
    }
}
